components/Skelet.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+const Skelet = () => {
+  const mockList = [1, 2, 3]
+
+  return (
+    <>
+      {
+        mockList.map((el : any) => (
+          <div key={`${el}-skelet`} className="w-[300px] flex flex-col border-2 rounded-lg border-gray-200 p-5 bg-white">
```

Comment:
reference here: https://flowbite.com/docs/components/skeleton/#image-placeholder

they have there `animate-pulse` className which gives more sense of loading.